src/node_buffer.cc

```diff
@@ -17,15 +17,15 @@
 
 #define MIN(a, b) ((a) < (b) ? (a) : (b))
 
-#define CHECK_NOT_OOB(r)                                                    \
-  do {                                                                      \
-    if (!(r)) return env->ThrowRangeError("out of range index");            \
+#define CHECK_NOT_OOB(r)                                                      \
+  do {                                                                        \
+    if (!(r)) return THROWI18NRANGEERROR(INDEX_OUT_OF_RANGE);                 \
   } while (0)
 
-#define THROW_AND_RETURN_UNLESS_BUFFER(env, obj)                            \
-  do {                                                                      \
-    if (!HasInstance(obj))                                                  \
-      return env->ThrowTypeError("argument should be a Buffer");            \
+#define THROW_AND_RETURN_UNLESS_BUFFER(env, obj)                              \
+  do {                                                                        \
+    if (!HasInstance(obj))                                                    \
+      return THROWI18NTYPEERROR(ARGUMENT_BUFFER);                             \
```

Comment:
think I see it in one of the define's above. mind if we always explicitly use it? it's really only for show (the `env` here could be removed and as long as it were set before this macro was called it would still work) but helps people like me follow the code easier. :)